### PR TITLE
Add Connection Test to TechCheck

### PR DESCRIPTION
**Pull request by joris974**
@AKurilin 

Removed Vimeo
Add Connection Check by downloading large picture

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -214,6 +217,23 @@
...
+      var endTime = new Date().getTime();
+      var duration = (endTime - startTime) / 1000;
+      if (duration > 5) {
+        displayCheckFailed(validatorId, null, "Your connection is slow. Your may experience slowness with Front Row.");
+      } else {
+        displayCheckSucceed(validatorId);
+      }
+    }
+    // This is a 3.4MB picture
+    picture.src =  assetsUrl + "/test.png" + "?n=" + Math.random();
```
Comment (cdparks) [inline js/main.js:233]: What does the `n` parameter do here?

Comment (joris974) [inline js/main.js:233]: change the URL so it is not in cache if you reload

Comment (cdparks) [inline js/main.js:233]: Ah, nice.

File: js/main.js [hunk 2]
```diff
@@ -340,8 +360,7 @@
     switch (validatorId) {
       case "getapi":      return "Fetching information from Front Row";
       case "patchapi":    return "Sending information to Front Row";
-      case "cloudfront":  return "Fetching information from Cloudfront";
-      case "s3":          return "Fetching information from S3";
+      case "frontrowcdn": return "Fetching information from Front Row Amazon";
```
Comment (cdparks) [inline js/main.js:363]: We can't use `flow` in this file?

File: js/main.js (hunk 1 above)
Comment (akurilin) [inline js/main.js:233]: crafty

**Comment by cdparks** [comment]
lgtm

File: js/main.js (hunk 1 above)
Comment (jdreaver) [inline js/main.js:233]: That's slick.

**Pull request merged**